src/api-client.ts

```diff
@@ -71,15 +84,20 @@ export abstract class ApiClient {
 
     console.log(`Before fetch URL: ${url}`);
     console.log(`Before fetch payload: ${requestOptions.payload}`);
-    const res = UrlFetchApp.fetch(url, requestOptions);
+    try {
+      const res = UrlFetchApp.fetch(url, requestOptions);
+      if (200 != res.getResponseCode() && 204 != res.getResponseCode()) {
+        Logger.log('HTTP code: ' + res.getResponseCode());
```

Comment:
Would you mind aligning to either console.log or Logger.log, please? Even though they are functionally equivalent, I think it keeps the code more readable if we align on one.